PWGCF/FEMTOSCOPY/AliFemto/AliAnalysisTaskFemto.cxx

```diff
@@ -77,21 +78,24 @@ AliAnalysisTaskFemto::AliAnalysisTaskFemto(TString name,
   f4DcorrectionsLambdas(NULL),
   f4DcorrectionsLambdasMinus(NULL),
   fGridConfig(aGridConfig),
-  fConfigTMacro(NULL),
+  fConfigTMacro(aMacro),
   fSaveConfigTMacro(NULL)
 {
   // Constructor.
   // Input slot #0 works with an Ntuple
   //DefineInput(0, TChain::Class());
   // Output slot #0 writes into a TH1 container
+  LoadMacro(fConfigTMacro);
```

Comment:
I thought I left a comment here, but I don't see it anymore.

Anyways, this would segfault if fConfigTMacro was NULL, so I fixed it by ignoring NULL values in LoadMacro (merge #11212)